src/app/shared/utils/account.ts

```diff
@@ -0,0 +1,5 @@
+import { User, Account } from '../models';
+
+export function isHaveSameAccount(user: User, account: Account) {
+  return user.account === account.user[0].account;
```

Comment:
Please make sure that user array is not empty

Reply:
API doc tells that account.user implies element has a child.
Do I need to write an additional check in the code?

Reply:
yes, and please, rename function to represent the meaning "User belongs to the account"